doc/Type/Attribute.pod6

```diff
@@ -10,8 +10,42 @@ In Raku lingo, an I<attribute> refers to a per-instance/object storage slot.
 An C<Attribute> is used to talk about classes' and roles' attributes at the
 metalevel.
 
-Normal usage of attributes does not require the user to use this class
-explicitly.
+C<Attribute> is typically useful when working with the L<MOP|/language/mop>.
+For instance, the names of all the attributes of a type can be introspected
+using its L<name|#method_name> method:
+
+=begin code :solo
```

Comment:
Why :solo? This would compile correctly, right? And it does not need its own file.